queries.sql

```diff
@@ -139,3 +139,85 @@ JOIN animals a ON o.id = a.owner_id
 GROUP BY o.id, o.full_name
 ORDER BY COUNT(a.id) DESC
 LIMIT 1;
+
+
+-- Write queries to answer the following:
+
+    --  Who was the last animal seen by William Tatcher?
+    SELECT a.name AS animal_name
+    FROM visits v
+    JOIN animals a ON v.animal_id = a.id
+    JOIN vets vt ON v.vet_id = vt.id
+    WHERE vt.name = 'William Tatcher'
+    ORDER BY v.visit_date DESC
+    LIMIT 1;
+
+  --  How many different animals did Stephanie Mendez see?
+    SELECT COUNT(DISTINCT a.id)
+    FROM visits v
+    JOIN animals a ON v.animal_id = a.id
+    JOIN vets vt ON v.vet_id = vt.id
+    WHERE vt.name = 'Stephanie Mendez';
+
+    --  List all vets and their specialties, including vets with no specialties.
+    SELECT vt.name, COALESCE(array_agg(s.name), ARRAY[]::text[]) AS specialties
+    FROM vets vt
+    LEFT JOIN vet_specialties vs ON vt.id = vs.vet_id
+    LEFT JOIN specialties s ON vs.specialty_id = s.id
+    GROUP BY vt.id, vt.name;
```

Comment:
Please add the screenshot of the query result 👀

![image](https://github.com/DevaWinner/vet-clinic/assets/78251927/c00bc22c-4c36-4113-a11e-031a542d61c6)

Reply:
Thank you for noticing the mix with the screenshots. I will sort them now